sfcasts/ep1/tdd.md

```diff
@@ -1,43 +1,55 @@
 # TDD
 
-All right. So one of the problems is that when Bob the park ranger sees the
+All right. So one of the problems is that when Bob, our park ranger, sees the
 dinosaur size... he can't remember if these are in meters... or centimeters...
 which makes a big difference when you step into a cage.
 
 A better way might be to just use words like small, medium, or large. So... let's
 do that!
 
+## What is TDD?
+
 *But*, to add this feature, we're going to use a philosophy called
-test driven development or TDD. TDD is basically a buzzword that describes a
+Test Driven Development or TDD. TDD is basically a buzzword that describes a
 4-step process for writing your test *before* the feature.
 
-Step 1: write a test for the feature. Step 2: run your test and watch it fail...
-since you haven't created that feature yet! Step 3: Write as little code as possible
-to get your test to pass. And Step 4: now that it's passing, refactor your code
-if needed to make it more awesome.
+Step 1: Write a test for the feature. Step 2: Run your test and watch it fail...
+since we haven't created that feature yet! Step 3: Write as little code as
+possible to get our test to pass. And Step 4: Now that it's passing, refactor
+your code if needed to make it more awesome
 
-So, to get the Small, Medium or Large text, I think we should add a new
+So, to get the Small, Medium, or Large text, I think we should add a new
 `getSizeDescription()` method to our `Dinosaur` class. *But*, remember, we're
-going to do this the TDD way, where step 1 is to write a *test* for that method...
-even though it doesn't exist yet. Yes, I know that's weird... but it's kinda awesome.
+going to do this the TDD way, where Step 1 is to write a *test* for that method...
+even though it doesn't exist yet. Yes, I know that's weird... but it's kinda awesome!
+
+## Step 1: Write a test for the Feature
 
 Add `public function` and let's first test that a dinosaur that's over 10 meters
 or greater is large. Inside, say `$dino = new Dinosaur()`, give him a name,
-let's use Big Eaty again, he's a cool dude, and set the length to 10.
+let's use Big Eaty again, since he's a cool dude, and set his length to 10.
 
-Then, `assertSame()` that `Large` will be equal to `$dino->getSizeDescription()`.
-For our failure message, lets use This is supposed to be a large Dinosaur.
+Then, `assertSame()` that `Large` will be identical to `$dino->getSizeDescription()`.
+For our failure message, let's use this is supposed to be a large Dinosaur.
```

Comment:
Minor: If we literally type "this is supposed to be a large dinosaur" then put it either in quotes or ticks and make sure the letters have the same upper case as what we type.